src/main/java/com/devonfw/quarkus/productmanagement/service/v1/ProductRestService.java

```diff
@@ -47,9 +47,6 @@
   @Inject
   UcManageProduct ucManageProduct;
 
-  @APIResponses({
-  @APIResponse(responseCode = "200", description = "OK", content = @Content(schema = @Schema(implementation = PagedProductResponse.class))),
-  @APIResponse(responseCode = "500") })
   @Operation(operationId = "Get Products", description = "Returns list of Products matching given criteria, uses pagination")
```

Comment:
We also don't need the operation doc. The doc should be represented by the javadoc of the interface and could directly been taken com servidedocgen

Reply:
removed "operation" annotations from ProductRestService.java